webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -470,6 +465,17 @@ def update_corr_param_resp(
         parent.save_state("active_correlation", corr_param_resp)
         return corr_param_resp
 
+    def get_selected_ensembles_from_store(
+        ensemble_selection_store: Dict[str, List]
+    ) -> Optional[List[str]]:
+        if ensemble_selection_store:
+            ensembles = [
+                selection["value"] for selection in ensemble_selection_store["selected"]
+            ]
+        else:
+            ensembles = None
+        return ensembles
```

Comment:
but then it's harder to print the value for, you know, for the print debugger! ; )

Reply:
Maybe this would be a solution in the middle :-) 

```
if ensemble_selection_store:
            ensembles = [selection["value"] for selection in ensemble_selection_store["selected"]]
            return ensembles
return None

```

Reply:
I think @valentin-krasontovitsch was joking there, but my suggestion was not a hard point, so @valentin-krasontovitsch  is free to do what he wants either keep his original version or use @hnformentin suggestion

Reply:
thanks for the suggestions, i can totally make it concise, it was mostly a joke, with a hint of truth in it, as my favorite way of debugging (in python) is indeed just printing variable values...

Reply:
made concise.